### PR TITLE
N-Triples  base direction tests.

### PR DESCRIPTION
**Pull request by gkellogg**
Positive and Negative tests for the use of base direction in N-Triples.

### CODE REVIEW

File: rdf/rdf12/rdf-n-triples/syntax/ntriples-base-1.nt [hunk 1]
```diff
@@ -0,0 +1 @@
+<http://example/a>  <http://example/b> "Hello@en--ltr" .
```
Comment (rubensworks) [inline rdf/rdf12/rdf-n-triples/syntax/ntriples-base-1.nt:1]: Should these looks as follows instead?
```suggestion
<http://example/a>  <http://example/b> "Hello"@en--ltr .
```

Comment (gkellogg) [reply rdf/rdf12/rdf-n-triples/syntax/ntriples-base-1.nt:1]: Yes, indeed.

File: rdf/rdf12/rdf-n-triples/syntax/ntriples-base-1.nt [hunk 2]
```diff
@@ -0,0 +1 @@
+<http://example/a>  <http://example/b> "Hello"@en--ltr .
```
Comment (rubensworks) [inline rdf/rdf12/rdf-n-triples/syntax/ntriples-base-1.nt:1]: Minor nit: there are two spaces between `<http://example/a>` and `<http://example/b>`.
Not wrong, but a single space might be a bit nicer.

**Review by rubensworks — APPROVED** [review]
Looks good to me!

I'm not able to test it on my end though. I have to do some implementation work first.

File: rdf/rdf12/rdf-n-triples/syntax/ntriples-base-1.nt (hunk 2 above)
Comment (gkellogg) [reply rdf/rdf12/rdf-n-triples/syntax/ntriples-base-1.nt]: Done.

**Review by Tpt — APPROVED** [review]

**Pull request merged**